llvm/cmake/config-ix.cmake

```diff
@@ -257,6 +257,35 @@ else()
   set(LLVM_ENABLE_TERMINFO 0)
 endif()
 
+#Check for icu.
+if(LLVM_ENABLE_ICU)
+  set(LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ".so")
+  if (LLVM_ENABLE_ICU STREQUAL FORCE_ON)
+    find_package(ICU REQUIRED COMPONENTS uc i18n)
+    if (NOT ICU_FOUND)
+      message(FATAL_ERROR "Failed to configure icu, but LLVM_ENABLE_ICU is FORCE_ON")
+    endif()
+  else()
+    find_package(ICU COMPONENTS uc i18n)
+  endif()
+  set(HAVE_ICU ${ICU_FOUND})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ${LIBRARY_SUFFIXES})
+endif()
+
+# Check for iconv.
+if(LLVM_ENABLE_ICONV)
+  if (LLVM_ENABLE_ICONV STREQUAL FORCE_ON)
+    find_package(Iconv REQUIRED)
```

Comment:
I'm concerned that on Windows, this might pull in GNU iconv, which is not license compatible with LLVM.

Reply:
Maybe we can only allow Iconv_IS_BUILT_IN to make sure its from C library on POSIX platforms? Or we would have to make sure the GNU one is dynamically linked as well similar to ICU

Reply:
If anyone is statically linking to libc, they're on their own in terms of licensing.

I'm specifically worried about Windows here because Microsoft libc doesn't have iconv, but mingw has a package for GNU iconv.  So the iconv license would be different from the libc license.

Reply:
I modified it now to only accept builtin iconv, so this should address the concern about accidentally picking up GNU iconv. Please let me know if this solves the Windows issue, thanks!